### ISSUE TITLE
Database import failed

### ISSUE THREAD

**Issue by hacksenwerk** [davidhealey/waistline#706]
I want to delete (not archive) some foods. 
I changed their name to `DELETE` and exported the database. 
I edited the exported database and removed the entries with `"name":"DELETE"` and the rest of that stuff that follows for that entry. 
But when I import that edited database I get: 
`Import failed`

**Comment by davidhealey** [comment]
Were those items referenced anywhere else, in the diary for example?

**Comment by hacksenwerk** [comment]
Yes, but that's why I edited the exported diary file too and delete them in there, but I can not import the edited diary file again.

**Comment by hacksenwerk** [comment]
I'm not confusing anything, how do you come to this? 
[In Issue 705](https://github.com/davidhealey/waistline/issues/705) @davidhealey suggested that I should try importing the diary file as a database file. 
I know what I'm doing!

**Comment by EmilJunker** [comment]
But you said that the diary export is not useful for humans because it only contains IDs. That's not true, so I assumed you were talking about the other file.

Anyway, the point is: If you make edits to the json file, you have to be careful not to mess up the syntax. Otherwise you won't be able to import the file again - which is exactly the issue you are having (i.e. the original topic of this issue).
